laws/src/main/scala/cats/laws/discipline/TraverseFilterTests.scala

```diff
@@ -47,6 +48,7 @@ trait TraverseFilterTests[F[_]] extends FunctorFilterTests[F] {
     EqFB: Eq[F[B]],
     EqFC: Eq[F[C]],
     EqGFA: Eq[Option[F[A]]],
+    EqGFB: Eq[Option[F[B]]],
```

Comment:
I don't see why we need this. Can you comment?

I don't think we can add this, but we can derive it:

```scala
implicit val EqOptFB: Eq[Option[F[B]]] =
  new Eq[Option[F[B]]] {
    def eqv(left: Option[F[B]], right: Option[F[B]]) =
      (left, right) match {
        case (Some(l), Some(r)) => EqFB.eqv(l, r)
        case (None, None) => true
        case _ => false  
      }
  }
```
and put it before `new DefaultRuleSet` below.

Reply:
you are right, this was not necessary.